lms/envs/common.py

```diff
@@ -2321,6 +2321,9 @@ def _make_locale_paths(settings):
     'csrf.apps.CsrfAppConfig',  # Enables frontend apps to retrieve CSRF tokens.
 
     'edly_panel_app',
+
+    # q-verse app for registration
+    'openedx.features.qverse_features.registration'
```

Comment:
Shouldn't we keep the app name generic enough so later on more changes can be done in the app?

Reply:
The main idea is that we will be having the main python package named `qverse_features` like in edx there is `openedx.features`.
In that package, we will introduce other apps. Here we are not restricted to only one app.

In UCSD we made the ucsd_features an app and now what we are doing just including new packages in it like ecommerce e.t.c.

Reply:
yes, this is exactly my point that we should be using 'openedx.features.qverse_features' in future instead of 'openedx.features.qverse_features.registration', so can we do this right from the beginning?

Reply:
discussed.

Reply:
whenever you discussed something offline, for reference, please add discussion here as well.